scripts/cfn

```diff
@@ -23,14 +23,32 @@ def upload_template_to_s3(conn, region, bucket_name, key_name, template):
         elif region == 'eu-west-1':
             region = boto.s3.connection.Location.EU
 
-        cloudformation_bucket = conn.create_bucket(bucket_name, location=region)
+        cloudformation_bucket = conn.create_bucket(bucket_name,
+                                                   location=region)
     key = cloudformation_bucket.new_key(key_name)
     key.set_contents_from_string(template)
     cfn_template_url = "https://s3.amazonaws.com/%s/%s" % (
         bucket_name, key_name)
     return cfn_template_url
 
 
+def validate_template(conn, template=None, url=None):
+    """Validate the template"""
+    print("Validating Template")
+    try:
+        if url:
+            conn.validate_template(template_url=url)
+        else:
+            conn.validate_template(template_body=template)
+    except boto.exception.BotoServerError as e:
+        # XXX - need to figure out why this isn't getting parsed from boto.
+        print("Error: %s" %
+              (literal_eval(e.error_message)['Error']['Message'],))
+        print("Exiting...")
+        sys.exit(1)
+    print("Template Validated")
```

Comment:
Good point about not wanting to loose the error message. 

Out of scope for this PR, but there is something wrong with the way the `logging` feature and `print` is used in this script over all, but I'm not sure what it is.